src/types.cpp

```diff
@@ -729,7 +729,7 @@ const char* _exvGettext(const char* str)
 
     if (!exvGettextInitialized) {
         //bindtextdomain(EXV_PACKAGE_NAME, EXV_LOCALEDIR);
-        const std::string localeDir = Exiv2::getProcessPath() + EXV_LOCALEDIR;
+        const std::string localeDir = EXV_LOCALEDIR[0] == '/' ? EXV_LOCALEDIR : (Exiv2::getProcessPath() + EXV_LOCALEDIR);
```

Comment:
Hmm, weird, somehow I have lost `"/" +` here when rebasing (it present in both places in the [original PR](https://github.com/Exiv2/exiv2/pull/1263).

Reply:
Should be fixed now.

Reply:
It is fixed, now.  Thank You very much.